docs/spec.md

```diff
@@ -319,11 +319,6 @@ in StableHLO programs. In the meanwhile, here is the list of these operations:
   `dynamic_gather`, `dynamic_iota`, `dynamic_pad`, `dynamic_reshape`,
   `real_dynamic_slice`, `set_dimension_size`
   ([#8](https://github.com/openxla/stablehlo/issues/8)).
-* "Quantization" category of StableHLO operations - they were bootstrapped from
-  MHLO, but we haven't specced them yet: `uniform_quantize`
-  ([#531](https://github.com/openxla/stablehlo/issues/531)) and
-  `uniform_dequantize`
-  ([#530](https://github.com/openxla/stablehlo/issues/530)).
```

Comment:
**update**

Agreed!

I note that, for `stablehlo.quantize`, the verification check for `C2` is missing and hence verification is `no`. For `stablehlo.dequantize`, both `C1` and `C2` are verified (via ODS and https://github.com/openxla/stablehlo/blob/a5bce5ea7ac348a26c685e614dd70160df617776/stablehlo/dialect/TypeInference.cpp#L2984 ), hence the verification and type-inference columns have a `yes`.

Reply:
I think uniform_quantize falls under "revisit", per guidance from status.md: "no: there is no implementation, but working on that is part of the roadmap. Note that Verifier can never be labeled as "no" because the ODS already implements some verification".

Reply:
Also, opened #1603 to track C2 for UniformQuantizeOp.

Reply:
Thanks!